### ISSUE TITLE
Errors on Set-DisplayRefreshRate

### ISSUE THREAD

**Issue by theking2** [MartinGC94/DisplayConfig#3]
# Set-DisplayRefreshRate -DisplayId 1 -RefreshRate 50
Set-DisplayRefreshRate: De configuratiegegevens voor dit product zijn beschadigd. Neem contact op met het ondersteunend personeel.

**Comment by MartinGC94** [comment]
Use the `AllowChanges` parameter like this: `Set-DisplayRefreshRate -DisplayId 1 -RefreshRate 50 -AllowChanges`. This allows Windows to make slight changes to the config so it can work with your display. This is needed because your display probably doesn't support 50 Hz exactly, but more like 49.951 or something similar.

As a side note, I think it's rude to post such a low effort question/issue. How can you expect other people to bother to help you when you won't provide any context or even translate the error message into English?

**Comment by gviehmann** [comment]
Usually translating error messages is not a good idea. I had the same problem with a german error message (Use-DisplayConfig : Die Konfigurationsdaten für dieses Produkt sind beschädigt. Wenden Sie sich an den Support).
This worked for me:
```
Get-DisplayConfig |
    Set-DisplayResolution  -Width 2560 -Height 1600 -DisplayId 1 |
    Set-DisplayRefreshRate -RefreshRate 60 -DisplayId 1 |
    Use-DisplayConfig -AllowChanges
Set-DisplayScale -Scale 200 -DisplayId 1
```

**Comment by MartinGC94** [comment]
Why would a translation be a bad idea? The localization process is typically handled by community helpers or by dedicated localization teams. It's very rare that it's the developers themselves, so if you are writing directly to the devs like you would on GitHub it's probably best to default to English unless the ReadMe tells you otherwise.

If I were to get a localized error message I'd post the exact error message along with a translation to English. That way it's possible for other people to find it with a verbatim search in Google while also telling the developers what the error message says.

In this case the error message has been localized by Windows and I'm just surfacing the exact error message from the underlying API.

Anyway, thanks for posting another solution for other users.

**Comment by MartinGC94** [comment]
What do you mean? We've already mentioned a way to get it working: Use `the `-AllowChanges` parameter. If that's not working for you then you need to provide more details.

**Comment by MartinGC94** [comment]
No response so I guess there's nothing else you need. I've released a new 2.0 version that includes a recommended action so if someone makes a mistake like this again the error tells them what to do. It looks like this:

```
PS C:\> Set-DisplayRefreshRate -DisplayId 1 -RefreshRate 123
Set-DisplayRefreshRate: Konfigurationsdataene for dette produkt er beskadiget. Kontakt en supporttekniker.
  Recommendation: Run the command again with the -AllowChanges parameter.
PS C:\>
```
Unfortunately it's only the latest preview versions of PowerShell that shows the recommended action by default. Users on older versions can check the recommended actions for errors like this: `$Error[0].ErrorDetails`.
